src/commands/spawn.h

```diff
@@ -1,472 +1,76 @@
-        "v016_car_armoredarticulatedtruck_rebel",
-        "v017_car_vintagemuscle_bomb_special",
-        "v017_car_vintagemuscle_civilian",
-        "v017_car_vintagemuscle_civilian_02",
-        "v017_car_vintagemuscle_rico",
-        "v018_car_monstertruck_bomb_special",
-        "v018_car_monstertruck_civilian_01",
-        "v019_car_modernlimo_bomb_special",
-        "v019_car_modernlimo_civilian_01",
-        "v020_car_moderngrandtourer_civilian_01",
-        "v021_car_sportsmuscle_bomb_special",
-        "v021_car_sportsmuscle_civilian_01",
-        "v022_car_moderncircuitracer_bomb_special",
-        "v022_car_moderncircuitracer_civilian_01",
-        "v023_car_racingsuper_bomb_special",
-        "v023_car_racingsuper_civilian_01",
-        "v023_car_racingsuper_racing_01",
-        "v024_car_ecosuper_bomb_special",
-        "v024_car_ecosuper_civilian_01",
-        "v024_car_ecosuper_racing_01",
-        "v025_car_vintagesuper_bomb_special",
-        "v025_car_vintagesuper_civilian_01",
-        "v026_car_vintagesports_bomb_special",
-        "v026_car_vintagesports_civilian_01",
-        "v027_car_hurricanetruck_civilian",
-        "v027_car_hurricanetruck_tesla",
-        "v027_car_hurricanetruck_transport",
-        // "v028_car_",
-        "v029_car_smallmodernsedan_civilian_01",
-        "v029_car_smallmodernsedan_civilian_02",
-        "v030_car_oldmini_civilian_01",
-        "v030_car_oldmini_civilian_02",
-        "v031_car_racinghothatch_civilian_01",
-        "v031_car_racinghothatch_encounter_takedown",
-        "v032_car_luxurysportssedan_civilian_01",
-        "v033_car_modernmini_civilian_01",
-        "v033_car_modernmini_civilian_02",
-        "v034_car_oldtruck_commercial_01",
-        "v034_car_oldtruck_commercial_cargo_barrel",
-        "v034_car_oldtruck_commercial_cargo_barrel_mission_special",
-        "v034_car_oldtruck_commerical_cargo_random",
-        "v034_car_oldtruck_commerical_cargo_sargento_intro",
-        "v035_car_modernvan_civic_ambulance_01",
-        "v035_car_modernvan_commerical_01",
-        "v035_car_modernvan_garland",
-        "v036_car_modernpickup_civilian_01",
-        "v036_car_modernpickup_civilian_02",
-        "v037_car_modernbus_commerical_01",
-        "v038_car_modernsuv_civilian_01",
-        "v038_car_modernsuv_civilian_02",
-        "v039_car_oldcampervan_civilian_01",
-        "v039_car_oldcampervan_civilian_02",
-        "v040_car_oldcompact_civilian_01",
-        "v040_car_oldcompact_sargento",
-        // "v041_car_",
-        "v042_car_racingsedan_civilian",
-        "v042_car_racingsedan_racing_01",
-        // "v043_car_",
-        // "v044_car_",
-        "v045_car_minetruck_commercial_01",
-        "v046_car_racingbuggy_civilian",
-        "v046_car_racingbuggy_military",
-        "v046_car_racingbuggy_racing_01",
-        // "v047_car_",
-        // "v048_car_",
-        // "v049_car_",
-        "v050_car_toyjeep_civilian",
+    struct PropertyFileEntry {
```

Comment:
```suggestion
#pragma pack(push, 1)
     struct PropertyFileEntry {
```
While it's not a huge issue, because you're using a pointer to this struct in `PropertyFileResult`, I think doing this now and preventing any bugs related to it in the future is a good idea.